src/stories/Introduction.mdx

```diff
@@ -0,0 +1,194 @@
+
+<div className='link-list'>
+  <a className='link-item' href='https://react.dev/' target='_blank'>
+    <img src={ReactImg} alt='React' />
+    <span>
+      <strong>React</strong>
+      Cinnamon is a library specifically designed for React projects, offering an extensive collection of components carefully tailored for seamless integration with React applications.
+    </span>
+  </a>
+  <a className='link-item' href='https://www.keycloak.org/' target='_blank'>
+    <img src={KeycloakImg} alt='Keycloak' />
+    <span>
+      <strong>Keycloak</strong>
+      We use Keycloak to implement access management. You can use it as well or use
+      your own solution.
+    </span>
+  </a>
+  <a
+    className='link-item'
+    href='https://mui.com/'
+    target='_blank'
+  >
+    <img src={MUIimg} alt='MUI' />
+    <span>
+      <strong>MUI</strong>
+     Cinnamon uses the MUI comprehensive suite of UI tools as the foundation for the components.
+    </span>
+  </a>
+  <a className='link-item' href='https://rollupjs.org/guide/en/' target='_blank'>
+    <img src={RollupImg} alt='Rollup' />
+    <span>
+      <strong>Rollup</strong>
+
+      Rollup is a module bundler for JavaScript which compiles small pieces of code
+      into something larger and more complex, such as a library or application.
+    </span>
+
+  </a>
+</div>
+
+<div className='subheading'>Repository information</div>
+
+<div className='link-list' >
+  <a className='link-item' href='https://github.com/CinCoders/cinnamon/blob/main/docs/CONTRIBUTING.md' target='_blank'>
+    <img src={Comments} alt='repo' />
+    <span>
+      <strong>Contributions are welcome!</strong>
+      To know a step by step guide on how to contribute
+      refer to our contribution guide. <br></br><br></br><br></br>
+      Show your ❤️ and support by giving a ⭐.
+    </span>
+  </a>
+  <a className='link-item' style={{height: '140px'}} href='https://www.npmjs.com/package/@cincoders/cinnamon' target='_blank'>
+     <div>
+     <div style={{display:'flex'}}>
+     <img src={InstallIcon} alt='Usage' />
+
+      <span>
+        <strong>Installation in other projects</strong>
+        To install the Cinnamon package into your project use the node package manager
+        and run the command:
+      </span>
+      </div>
+      ```bash
+       npm install @cincoders/cinnamon
+       ```
+      </div>
+
+  </a>
+  <a
+    className='link-item'
+    href='https://github.com/CinCoders'
+    target='_blank'
+    style={{height: '80px'}}
+  >
+    <img src={GitImg} alt='code' />
+    <span>
+      <strong>GitHub project</strong>
+      View the source and add issues.
+    </span>
+  </a>
+  <a
+    className='link-item'
+    href='https://github.com/CinCoders/cinnamon/blob/main/LICENSE'
+    target='_blank'
+    style={{height: '80px'}}
+  >
+    <img src={LicenseImg} alt='comments' />
+    <span>
+      <strong>MIT License</strong>
+      Terms and conditions for the use and distribution of copyrighted works.
```

Comment:
Please use this text to explain the license:
CInnamon is released under the MIT license, which means that you have the freedom to use, modify, and distribute the package for any purpose. The MIT license promotes openness and collaboration in software development, allowing you to integrate CInnamon into your projects with ease. See MIT License for more details.